contents.md

```diff
@@ -33,16 +33,44 @@ at home to participate offline stage. Learn more in [README](README.md).
 
 ## Stage 1. The Show Must Go On
 
-__Under revision__
+### Main thread
 
+ 1. Intro to meet-up backed course stage
+ 1. Self-study stage practical tasks retrospection
  1. Mastering HTML 5 forms
- 1. Responsive Layouts
- 1. Backend API (Weather App)
- 1. Pizza App: Login and Registration
- 1. Pizza App: Pizza Creation
- 1. ReactJS Weather App
- 1. React + Redux
+ 1. Modern Web App and Web  Frameworks
+ 1. Component-based App Architecture (Weather App)
+ 1. Routing (Collection Browsing App)
+ 1. Authentication and Authorization (Chat App)
+ 1. Websockets (Chat App)
+ 1. Canvas API (Puzzle Game)
+ 1. Framework/Library (_TBD_)
+ 
+### Supplementary workshops
```

Comment:
We will share this description with students, so it will be good to provide a description of what "Supplementary" is.

As well as "Soft skill extras". This is also supplementary stuff, right?

Reply:
You are right. I had also an idea of describing these parts like "Supplementaries and extras are pools of topics to pick from as required by main thread and as per students' interest and to the extent the course time allows".

But something has stopped me from adding anything alike at this stage :)

My suggestion is still to improve this part of the document incrementally.

Reply:
At the moment when we merge this article, we will start to give the link to students. In general, it is about adding one description. I will add it by myself today if you don't mind of course.